packages/amplify_datastore/test/amplify_datastore_delete_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:amplify_datastore/amplify_datastore.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import './testData/Post.dart';
```

Comment:
In the Android library, we have this type of content in a module called `testmodels`. As the most minor of tweaks, you could rename `testData` to `test_models`.

Reply:
done.